app/controllers/KnolxUserAnalysisController.scala

```diff
@@ -0,0 +1,82 @@
+package controllers
```

Comment:
Make separate rest end points for didNotAttendCount, banCount, totalMeetUps, totalKnolx.
And a single rest end point for SessionsInformation.